lib/graphql/schema_comparator/changes/breaking.rb

```diff
@@ -0,0 +1,286 @@
+module GraphQL
+  module SchemaComparator
+    module Changes
+      class TypeRemoved < AbstractChange
+        attr_reader :removed_type, :criticality
+
+        def initialize(removed_type)
+          @removed_type = removed_type
+          @criticality = Changes::Criticality.breaking
+        end
+
+        def breaking?
```

Comment:
Two related things:

1. maybe just define this on `AbstractChange`? I believe it can always be determined via `@criticality` which can be the default implementation.
2. it would be nice to have `breaking?`, `dangerous?`, and `safe?` all exist on every change

Reply:
1. Agreed! good point.

2. Yep thats the goal! every change will have these methods that will just delegate to its criticality.